test/test_rb.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+A Tester for the RB experiment
+"""
+
+from qiskit.quantum_info.operators.predicates import matrix_equal
+from qiskit.quantum_info import Clifford
+from qiskit.quantum_info.operators.measures import process_fidelity
+from qiskit.test import QiskitTestCase
+from qiskit.test.mock import FakeParis
+from qiskit.exceptions import QiskitError
+import numpy as np
+import qiskit_experiments as qe
+
+
+class TestRB(QiskitTestCase):
+    """
+    A simple and primitive backend, to be run by the RB tests
```

Comment:
This is an incorrect description - you are describing the `TestRB` class, which is a class of tests, not a backend (also we don't call backends "primitive"!)

Reply:
Changed the description to a better one.